### DISCUSSION TITLE
Is there any way to get around the Mage Skull meditation glitch without having to turn off turbo mode?

### DISCUSSION THREAD

**Discussion by scroopsy** [Tobi-Mob/Skul.Mod#30] [Q&A]
I want to pick mage a lot but know that if I do I'll have to mash on my secondary skull. Should I do something like macro F2 to a random button on my controller? Any sort of work-around would be appreciated!

**Comment by Samvdg** [comment]
What do you mean exactly? Turbo button messing you up on mage?
If so, i'd def recommend mapping the turbo button to a button on your controller. I'd say L3 or R3 since i believe they're not used

**Reply by scroopsy** [reply]
When you try to meditate with the mage on turbo, the mage freaks out and it just doesn't work. But yeah, I actually did end up doing this eventually and mapped it to R3 like you said, it's a decent workaround as long as you remember to turn turbo back on when swapping off of mage.
